ccmlib/node.py

```diff
@@ -1349,9 +1349,11 @@ def _save(self):
         self.__update_envfile()
         self._update_config()
 
-    def _update_config(self):
+    def _update_config(self, on_stop=False):
         dir_name = self.get_path()
         if not os.path.exists(dir_name):
+            if on_stop:
+                return
             os.mkdir(dir_name)
```

Comment:
won't be easier to move the os.mkdir to a better place ? i.e. not part of update_config ?

Reply:
maybe, it'd cleaner, but it might be a bigger change.
I'll look into it.